meta/runtime.yml

```diff
@@ -21,7 +21,7 @@ plugin_routing:
     openshift:
       redirect: community.okd.openshift
     k8s:
-      deprecation:
+      tombstone:
         removal_version: 6.0.0
         warning_text: >-
           The k8s inventory plugin has been deprecated and
```

Comment:
I am getting the following sanity test error:
```
ERROR: Found 1 runtime-metadata issue(s) which need to be resolved:
ERROR: meta/runtime.yml:0:0: The tombstone removal_version ('6.0.0') must not be after the current version for dictionary value @ data['plugin_routing']['inventory']['k8s']['tombstone']['removal_version']. Got '6.0.0'
```
Do we need to wait until after 6.0.0 is release before we change `deprecation:` to `tombstone:`?

Reply:
To have it passed, you need to put a version like `6.0.0-dev0` in the galaxy.yaml as I did in the #867

Ideally, to complete [ACA-2407](https://issues.redhat.com/projects/ACA/issues/ACA-2407) and as the preparation to [ACA-2406](https://issues.redhat.com/projects/ACA/issues/ACA-2406) I recommend following:
- merge https://github.com/ansible-network/github_actions/pull/171
- merge https://github.com/ansible-network/github_actions/pull/169
- rebase to #867 and merdge.

/att @gravesm as the reporter of ACA-2407 and one of repo maintainer.

Reply:
@yurnov thanks so much for listing this all out. The two `github_actions` PRs have been merged; there is a merge conflict in #867, please let me know if you need any help with it. 

Once we get the conflict sorted out and the CI re-run, we can take another look at that PR.